plugins/org.komodo.relational/src/org/komodo/relational/model/Schema.java

```diff
@@ -21,6 +21,7 @@
  */
 package org.komodo.relational.model;
 
+import org.komodo.relational.RelationalObject;
 import org.komodo.spi.KException;
```

Comment:
It is because the interface that these interfaces extend was refactored to a different package. In my comment at the very top I try to explain the rationale for the refactoring.